### DISCUSSION TITLE
Billing (Type  2) NPI instead of TIN

### DISCUSSION THREAD

**Discussion by skochin** [CMSgov/price-transparency-guide#316] [Q&A] [answered]
We're seeing some situations where a provider group contract is based on a billing NPI (Type 2 NPI) instead of the group's Tax ID (EIN).  The same business entity (EIN) may have multiple contracts under different billing NPIs.  Would it be better to use the "NPI" type for the provider group's "tin" object (this would be the most accurate representation), or list multiple rates for each service under the same TIN? The "NPI" type is meant to signify a provider's personal NPI, so would a third value such as "NPI2" be appropriate?  For example (this is dummy data):

{
            "provider_group_id": 10050,
            "provider_group": {
                "npi": [
                    1041960114,
                    3628729440,
                    5319651146,
                    6237980620,
                    7177504496,
                    7590416438,
                    7872138074,
                    9142350324
                ],
                "tin": {
                    "type": "NPI2",
                    "value": "2351798415"
                }
            }
        },

**Comment by shaselton-usds** [comment, marked as answer]
At the moment the schema requires the TIN/NPI (type 1 or 2) combination to uniquely identify the negotiated arrangement.
